### ISSUE TITLE
Pretrained model link

### ISSUE THREAD

**Issue by Laqwad** [gangsterless/pytorch-softmax-splatting#7]
Hello, may I have another link for the pretrained model available outside China please, it's very difficult to get access to Baidu in Europe, ty.
